tests/util/yaml/test_init.py

```diff
@@ -67,17 +68,21 @@ def test_simple_dict(try_both_loaders):
     assert doc["key"] == "value"
 
 
-def test_unhashable_key() -> None:
+@pytest.mark.parametrize(
+    "yaml_configuration_files", [{YAML_CONFIG_FILE: "message:\n  {{ states.state }}"}]
```

Comment:
Since you are only adjusting the main config file, maybe this is sufficient?
```suggestion
    "yaml_configuration", ["message:\n  {{ states.state }}"]
```